DS_and_Algorithm/Selenium/test.java

```diff
@@ -0,0 +1,706 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import org.openqa.selenium.support.ui.Select;
+import org.openqa.selenium.JavascriptExecutor;
+
+public class SeleniumExample {
+
+    public static void main(String[] args) {
+        // Setup WebDriver
+        Systm.setProperty("webdriver.chrome.driver", "path/to/chromedriver");
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/Amruta101998/DS-and-Algorithm/pull/15#issuecomment-2180991075>#0bf007</a><span> - 06/20/2024, 03:35 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The method 'Systm.setProperty' is a typo and should be 'System.setProperty'. This will cause a runtime error as 'Systm' is not a recognized class.
</div>
<div id="fix">
<b>Fix:</b> Correct the typo from 'Systm' to 'System'.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -15,7 +15,7 @@
  public static void main(String[] args) {
      // Setup WebDriver
 -    Systm.setProperty("webdriver.chrome.driver", "path/to/chromedriver");
 +    System.setProperty("webdriver.chrome.driver", "path/to/chromedriver");
      WebDriver driver = new ChrmDriver();
```
</div>
</details>
</div>
<br>


</div>